### DISCUSSION TITLE
Does anyone know how to change the default base url of the jsoncrack service?

### DISCUSSION THREAD

**Discussion by peterwoo339** [AykutSarac/jsoncrack.com#121] [Q&A] [answered]
Hi everyone, I'm trying to deploy jsoncrack behind a location like "/jsoncrack" on nginx but the default base url is "/" for jsoncrack. Does anyone know how to change the code or any tricks to make this happen? Thanks!

**Comment by Cistalvo** [comment]
I agree that any guidance on how to set up jsoncrack behind a reverse proxy would be greatly appreciated.

**Reply by IvoLeist** [reply, marked as answer]
I am by far no nginx expert but something along that lines (below) should work, no (?):
*untestested:

```
server {
    listen 80;
	access_log  /var/log/nginx/http.access.log;
    
    location / {
        access_log  /var/log/nginx/http.access.log;
        error_log  /var/log/nginx/http.error.log;

        root /usr/share/nginx/html;
        index index.html index.htm;
        try_files $uri $uri/ /index.html =404;
    }
    
    location /jsoncrack/ {
        access_log /var/log/nginx/jsoncrack.access.log;
        error_log /var/log/nginx/jsoncrack.error.log;
        proxy_pass http://jsoncrack:8080/jsoncrack/;
        include includes/proxy-set-header.conf;
    }
}
```

In the example above nginx + jsoncrack would be run via Docker-compose

I make our jsoncrack accessible behind a reverse proxy using:
https://nginxproxymanager.com/
https://github.com/NginxProxyManager/nginx-proxy-manager

Maybe this makes it more easy for you as well :v:
